### ISSUE TITLE
Update partner logos to Vectors

### ISSUE THREAD

**Issue by samtaggart** [threefoldfoundation/www_cloud_threefold.io#62]
The logos on the current partner page are pixelated in higher resolutions and take unnecessary load time.

**Deliverable**
- [ ] Convert existing images to SVG (@samtaggart)
- [ ] Replace logo image files with SVGs (@ehab-hassan)
- [ ] Place files in a folder for future usage (@samtaggart)

**Comment by samtaggart** [comment]
@ehab-hassan @samaradel Tagged you both, not sure who will handle. Thanks!

**Comment by samtaggart** [comment]
- [ ] For Covid Fighters, please use this logo:
![logo_final-768x128](https://user-images.githubusercontent.com/50134479/89195748-7e195480-d5a9-11ea-9f67-fb8d28baa90b.png)

**Comment by samtaggart** [comment]
Please hold on:

- [ ] Artheon
- [ ] Zetako
- [ ] & Social, for now

**Comment by samtaggart** [comment]
- [ ] For Artheon, please use this logo:
![artheon_logo](https://user-images.githubusercontent.com/50134479/89393510-30b4f880-d70b-11ea-9bb7-668b86c52952.png)

**Comment by ehab-hassan** [comment]
I don't think we can find all logo SVG(vector) ,what is the problem with the logos we have now as PNG?
